refinery/config/settings/base.py

```diff
@@ -577,3 +578,5 @@ def get_setting(name, settings=local_settings):
 REFINERY_TUTORIAL_STEPS = refinery_tutorial_settings
 
 ANONYMOUS_USER_ID = -1
+
+TEST_RUNNER = "django.test.runner.DiscoverRunner"
```

Comment:
@hackdna: After reading up, `DiscoverRunner` is providing the same functionality that we are currently using Django nose for.

Some of django-nose's other notable features that we would lose are:
```
- Fixture bundling, an optional feature which speeds up your fixture-based tests by a factor of 4
- Reuse of previously created test DBs, cutting 10 seconds off startup time
- Hygienic TransactionTestCases, which can save you a DB flush per test
```

I don't have a strong opinion for the keeping or removal of django-nose.